src/index.js

```diff
@@ -4,6 +4,11 @@ import './index.scss';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 import { HashRouter as Router } from 'react-router-dom';
+import 'core-js/modules/es7.array.includes';
```

Comment:
One suggestion, these `core-js` polyfills for IE11 may need to be at the top of the file, so the polyfills are in place before App.js runs.